low-level/sensors/impl/generic/raid_integrity_data.py

```diff
@@ -103,15 +107,22 @@ def initialize(self, conf_reader, msgQlist, product):
         self._rack_id = self._conf_reader._get_value_with_default(
                                 self.SYSTEM_INFORMATION, COMMON_CONFIGS.get(self.SYSTEM_INFORMATION).get(self.RACK_ID), '001')
         self._node_id = self._conf_reader._get_value_with_default(
-                                self.SYSTEM_INFORMATION, COMMON_CONFIGS.get(self.SYSTEM_INFORMATION).get(self.NODE_ID), '001')  
+                                self.SYSTEM_INFORMATION, COMMON_CONFIGS.get(self.SYSTEM_INFORMATION).get(self.NODE_ID), '001')
         self._timestamp_file_path = self._conf_reader._get_value_with_default(
                                     self.RAIDIntegritySensor, self.TIMESTAMP_FILE_PATH_KEY, self.DEFAULT_TIMESTAMP_FILE_PATH)
         self._polling_interval = int(self._conf_reader._get_value_with_default(
                                 self.RAIDIntegritySensor, self.POLLING_INTERVAL, self.DEFAULT_POLLING_INTERVAL))
+        self._next_scheduled_time = self._polling_interval
+        self._fault_accepted_time = int(self._conf_reader._get_value_with_default(
```

Comment:
Please make sure to read this newly added key-value from /etc/sspl.conf file, 
because it will not be available in consul  as we didn't added it in sspl.sls file.

Reply:
oh good catch @shriya-deshmukh , that case add a seperate PR for cortx-provisioner too using cortx-1.0 branch, to update sspl.sls

Reply:
won't be required if we don't want to send 2nd alert after 1.2 days of fault in raid_integrity sensor.
this variable will be removed.

Reply:
removed.